internal/pkg/controller/controller.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stakater/Xposer/internal/pkg/constants"
 	"github.com/stakater/Xposer/internal/pkg/ingresses"
 	"github.com/stakater/Xposer/internal/pkg/routes"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
is `v1` required?

Reply:
Yes. VSCode was removing it automatically. So it had to be put like that